assessments/projects/volcano/threat-modelling.md

```diff
@@ -0,0 +1,59 @@
+# Lightweight Threat Modelling
+## Threat Modelling Notes
+- There are a total of 190 contributors and only 6 maintainers and owners thus there are 184 non-maintainer users who are in the working groups.
```

Comment:
I partially get what you're alluding to, but it's not helpful to mention this, as being a maintainer is challenging to begin with, and the team (although small) is committed and very talented!

Reply:
I see. If it won't be helpful, it might be best to remove it.
Removed it from the notes and recommendations section.